source/manual/data-gov-uk-deployment.html.md

```diff
@@ -75,6 +75,14 @@ Builds can take up to 10 minutes. You can view progress in GitHub Actions under
 
 Deployments typically take 5 minutes. You can view progress in Argo CD.
 
+### How to check that your release has been successfully deployed
+
+In order to validate that your release has been deployed you will need to check the relevant pod in the relevant environment for Argo CD.
```

Comment:
I'm not sure 

```suggestion
In order to validate that your release has been deployed, you will need to check the status of the relevant pod in the appropriate environment in Argo CD.
```

Reply:
what do you mean by you are not sure?

Reply:
I meant to delete that started sentence. My first thought was that this sentence was very generic, didn't really say what needed checking (there's a lot of info in Argo).